### ISSUE TITLE
Something on replit I've been working on

### ISSUE THREAD

**Issue by hello-smile6** [webdev03/webdev03#12]
https://crosh-backend.9pfs.repl.co/
https://replit.com/@9pfs/crosh?v=1
https://replit.com/@9pfs/crosh-backend?v=1
Next, I'm going to try using hterm for the frontend.

**Comment by hello-smile6** [comment]
@webdev03

**Comment by webdev03** [comment]
Cool! I wonder if you can get the dbus-send to somehow work.

**Comment by hello-smile6** [comment]
> Cool! I wonder if you can get the dbus-send to somehow work.

Nope, not without running debian in qemu. dbus doesn't work. If you go to https://chromium.googlesource.com/apps/libapps/+/refs/heads/main/hterm/doc/embed.md you can see the hterm docs.

**Comment by hello-smile6** [comment]
https://bash-server.9pfs.repl.co/ uses hterm, try running nano or byobu. Also, if you resize the window, it adjusts on the remote end.
